table/block_based_table_reader.cc

```diff
@@ -253,7 +257,8 @@ class PartitionIndexReader : public IndexReader, public Cleanable {
               index_value_is_full_),
           index_block_->NewIterator<IndexBlockIter>(
               icomparator_, icomparator_->user_comparator(), nullptr,
-              kNullStats, true, index_key_includes_seq_, index_value_is_full_));
+              kNullStats, true, index_key_includes_seq_, index_value_is_full_,
+              false /* block_contents_pinned */));
```

Comment:
Is the default value false? If yes, should we skip passing false?

Reply:
What't the benefit do you think if we do that? The code is shorter? I explicitly pass the argument so that people see it's not pinned.
Saying that, I can change if you think that's the proper way. It's not a big deal.

Reply:
Your way is better.